packages/quip-cli/src/commands/login.ts

```diff
@@ -175,6 +175,12 @@ export default class Login extends Command {
         const { site, force, hostname, port, config } = flags;
         const accessToken = flags["with-token"];
 
+        // displays error message if command has part as "--with-token=" without passing a value.
+        if (typeof accessToken !== 'undefined' && accessToken === "") {
```

Comment:
i think `if (accessToken === "")` should suffice. `===` guarantees that access token is not undefined